sdk/eventhub/azure-eventhubs/README.md

```diff
@@ -206,6 +207,54 @@ finally:
     pass
 ```
 
+### Consume events from all partitions of an Event Hub
```

Comment:
As suggested in the README for Javascript and .Net, I would suggest to change this as below

```suggestion
### Consume events using an Event Processor
```

The idea is to catch the user's attention on how this is different from the other sample on consuming events.

Reply:
thanks for the suggestion, doc is updated.